ts/Core/Renderer/SVG/SVGLabel.ts

```diff
@@ -434,7 +429,7 @@ class SVGLabel extends SVGElement {
             bBox.height || Infinity
         );
 
-        if (this.needsBox) {
+        if (this.needsBox && (!this.renderer.styledMode || this.text.added)) {
```

Comment:
Maybe I should just get rid of it and move the crisping outside the `!this.box` block instead so that it handles `stroke-width` changes as well?

Reply:
Yes I still don't understand the case, but it seems wrong to add more to the condition when `this.needsBox` is itself a condition.

Reply:
The case is that `stroke-width` needs to be known for crisping to work, and in styled mode you dont know until the text is added to the DOM, so I added that condition since the crisping was set only when creating the box. And this problem was discovered when I made the button width actually update when setting padding.